modules/local/simpleaf_index.nf

```diff
@@ -0,0 +1,48 @@
+process SIMPLEAF_INDEX {
+    tag "$transcript_gtf"
+    label "process_medium"
+
+    conda (params.enable_conda ? 'bioconda::simpleaf=0.5.1' : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/simpleaf:0.5.1--h9f5acd7_0' :
+        'quay.io/biocontainers/simpleaf:0.5.1--h9f5acd7_0' }"
+
+    input:
+    path genome_fasta
+    path transcript_fasta
+    path transcript_gtf
+
+    output:
+    path "salmon/index"              , emit: index
+    path "salmon/ref/*_t2g_3col.tsv" , emit: transcript_tsv
+    path "versions.yml"              , emit: versions
+    path "salmon"
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def seq_inputs = (params.transcript_fasta) ? "--refseq $transcript_fasta" : "--gtf $transcript_gtf"
+    """
+    # export required var
+    export ALEVIN_FRY_HOME=.
+
+    # prep simpleaf
+    simpleaf set-paths
+
+    # run simpleaf index
+    simpleaf \\
+        index \\
+        --threads $task.cpus \\
+        --fasta $genome_fasta \\
+        $seq_inputs \\
+        $args \\
+        -o salmon
+    
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        simpleaf: 0.4.0
```

Comment:
should not be hardcoded

Reply:
Yes, I forgot to add the command. Should not be hardcoded, indeed. But, we have a problem, the tool does not have a version command.

Reply:
I see! In general, we store the version in a variable in that case, see end of this section:
https://nf-co.re/developers/modules#general

Or maybe just output the version of salmon instead? The wrapper seems of minor importance.

Reply:
If we go for salmon, salmon is being grabbed in the quantification step already :)

Reply:
switched so that both simpleaf modules have simplef version in variable as you showed and also get salmon version